<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadioWindow</class>
 <widget class="QWidget" name="RadioWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>202</y>
     <width>478</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>KILÉPÉS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volumeUpButton">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>138</width>
     <height>98</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VOL+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volumeDownButton">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>101</y>
     <width>138</width>
     <height>99</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VOL-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeButton">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>1</y>
     <width>138</width>
     <height>98</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Csatorna
váltás</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="radioEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>1</y>
     <width>199</width>
     <height>98</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>25</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="channel1Button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>101</y>
     <width>168</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="channel2Button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>101</y>
     <width>169</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="channel3Button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>151</y>
     <width>168</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="channel4Button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>151</y>
     <width>169</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
